components-api/src/main/scala/pl/touk/nussknacker/engine/api/typed/AssignabilityDeterminer.scala

```diff
@@ -223,6 +226,19 @@ object AssignabilityDeterminer {
 
   }
 
+  private object NoConversionConversionChecker extends ConversionChecker {
```

Comment:
This name is unclear to me, why 'conversion' is repeated? What about `WithoutConversionChecker` is not good enough?